ui/e2e/package.json

```diff
@@ -14,10 +14,11 @@
     "lint:fix": "npm run lint -- --fix"
   },
   "devDependencies": {
+    "@perses-dev/prometheus-plugin": "0.21.1",
     "@playwright/test": "^1.29.1",
     "chokidar-cli": "^3.0.0",
     "eslint-plugin-playwright": "^0.11.2",
-    "happo-e2e": "^2.1.0",
+    "happo-e2e": "^2.1.1",
```

Comment:
We need this new version because there was a bug in the canvas handling. In positive news, they had it fixed within about an hour of me emailing support, so it's really nice to see how responsive they are to us, even in a "free" use case.